.storybook/main.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
     // Alternately, for an alias:
     config.resolve.alias['@bcrs-shared-components/enums'] = path.resolve(__dirname, '../') + '/src/enums';
     config.resolve.alias['@bcrs-shared-components/interfaces'] = path.resolve(__dirname, '../') + '/src/interfaces';
+    config.resolve.alias['@bcrs-shared-components/corp-type-module'] = path.resolve(__dirname, '../') + '/src/modules/corp-type-module';
```

Comment:
I had to make this specific because I originally didn't package "all modules" together -- just this one module.
Storybook needs this alias BTW.